src/__tests__/middy.js

```diff
@@ -276,7 +276,7 @@ describe('🛵  Middy test suite', () => {
     })
   })
 
-  test('If theres an error and one error middleware handles the error, the next error middlewares is executed', (endTest) => {
+  test('If theres is an error and one error middleware handles the error, the next error middlewares should not be executed', (endTest) => {
```

Comment:
Should there be `there` instead of `theres`?

Reply:
This was merged while inspecting the code, ignore it

Reply:
I did mess up with this one a little. I will do another commit to properly release this on 0.x and also fix this typo. So totally valuable comment :) Thanks

Reply:
(i will also fix in the port to 1.0)